lib/fog/aws/requests/data_pipeline/create_pipeline.rb

```diff
@@ -24,14 +24,35 @@ def create_pipeline(unique_id, name, description=nil, tags=nil)
             :body => Fog::JSON.encode(params),
             :headers => { 'X-Amz-Target' => 'DataPipeline.CreatePipeline' },
           })
-
-          Fog::JSON.decode(response.body)
```

Comment:
Works for me, I think. My only concern would be about how weird it might be (or not) for fog-core to be in a different major release level. They are distinct projects now, so probably fine, but just something to note.